src/PluginCreator.ts

```diff
@@ -148,7 +148,7 @@ export class PluginCreator {
             : module
     }
 
-    createTransformers: (TransformerHost) => ts.CustomTransformers = (main: TransformerHost) => {
+    createTransformers(main: TransformerHost) {
```

Comment:
In some cases i need to deletegate createTransformers method, for example rollup-plugin-typescript2.